utils/react/ReactUtils.js

```diff
@@ -30,6 +30,8 @@ export function getReactElementName(obj) {
  *      These will override any existing props placed on the element, and should be used with care.
  */
 export function elementFromContent(content, addProps) {
+    if (isNil(content)) return null;
```

Comment:
This is not necessary for this grid editors change, but it seemed reasonable.
It also did not enable any cleanups inside hoist-react.